jdaviz/configs/cubeviz/plugins/moment_maps/tests/test_moment_maps.py

```diff
@@ -24,4 +22,11 @@ def todo_fix_test_moment_calculation(spectrum1d_cube):
 
     assert mm.moment_available
     assert dc[1].label == 'Moment 0: test'
-    assert dc[1].get_object(cls=Spectrum1D, statistic=None).shape == (4, 2, 2)
+
+    # FIXME: Moment output has no WCS.
+    # Error from glue_astronomy/translators/spectrum1d.py to_object()
+    with pytest.raises(TypeError, match=r'data\.coords should be an instance of WCS'):
+        assert dc[1].get_object(cls=Spectrum1D, statistic=None).shape == (4, 2, 2)
```

Comment:
I am not sure why the original test is expecting a cube back. Zeroth moment is the collapsed sum, so it should have been 2D.

Also, do you really expect the output of the moment map plugin to have WCS?